### DISCUSSION TITLE
请问如何启动纯前端项目？

### DISCUSSION THREAD

**Discussion by vaynevayne** [illacloud/illa-builder#3501] [Q&A]
想学习源码，请问如何在本地启动该项目？ 谢谢

**Comment by AruSeito** [comment]
https://github.com/illacloud/illa-builder/blob/beta/CONTRIBUTING.md

@vaynevayne 非常抱歉，这么晚才回复您。您可以参考该文件

**Reply by vaynevayne** [reply]
@AruSeito 你好,还是不行,  git submodule 是不是没生效啊, 我记得这个命令好像会有一个modules的文件夹生成

<img width="677" alt="image" src="https://github.com/illacloud/illa-builder/assets/124666577/66bc9bd1-5305-4196-a66f-fff030f6a863">

**Reply by AruSeito** [reply]
git submodule init 看起来没生效

**Reply by vaynevayne** [reply]
嗯，能否加你微信

**Reply by AruSeito** [reply]
可以分开执行试一下。

**Reply by vaynevayne** [reply]
同样的问题
<img width="704" alt="image" src="https://github.com/illacloud/illa-builder/assets/124666577/ec3c1518-0b5f-4394-bef5-0ba4756c69f9">

**Comment by vaynevayne** [comment]
能否重新打开这个帖子？@AruSeito

**Reply by AruSeito** [reply]
您好，我们后续又更新了https://github.com/illacloud/illa-builder/blob/beta/CONTRIBUTING.md 启动流程。
您可以参考该流程进行操作。如果遇到子包等初始化失败，需要您自己查看为何无法初始化，例如：网络问题等。

**Reply by vaynevayne** [reply]
<img width="915" alt="image" src="https://github.com/illacloud/illa-builder/assets/124666577/2cbf2be4-31bc-4ccc-b3c1-e26c059e710a">
<img width="619" alt="image" src="https://github.com/illacloud/illa-builder/assets/124666577/91d86f86-31c0-4e05-9dc8-69dfe56dda9a">

子包里没这个文件, 会不会不是子包的问题
